frontend/src/exams/buttons-exams.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div>
+    <b-button class="btn-primary" @click="clickAddIndividual">Add Individual ITA Exam</b-button>
+    <AddIndividualItaExam />
```

Comment:
Are there going to be singular components for each method of adding exams, or do you imagine we'll be able to have some code re-use across them?

Reply:
well, for the ita there are only two types of entries (group or individual) and i'm leaning towards one component for each type.  is that okay?  they will both utilize the same parameter in the store (state.capturedExam) to hold their values before posting and use the same methods, but it makes sense to me to keep them separate.

Reply:
just because their templates are already quite long and pretty complex and since the fileds are quite different it the conditional parts would be most of the template and therefor 2 cases would almost double the amount of template per component and i think that is probably too long for a single component.